crates/gpui/examples/image.rs

```diff
@@ -0,0 +1,64 @@
+use gpui::*;
+
+struct ImageFromResource {
```

Comment:
We should derive `IntoElement` here so we can render this struct directly.

```suggestion
#[derive(IntoElement)]
struct ImageFromResource {
```